.github/workflows/manual_release.yml

```diff
@@ -1,31 +1,48 @@
-name: CD
+name: Manual Release
 
 on:
-  push:
-    branches: [main]
-
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
 jobs:
+  release:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.release.outputs.release }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set release
+        id: semrel
+        uses: go-semantic-release/action@v1
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          allow-initial-development-versions: true
+          force-bump-patch-version: true
+
+      - name: Output release
+        id: release
+        run: echo "::set-output name=release::${{ steps.semrel.outputs.version }}"
   publish_javascript:
     # The type of runner that the job will run on
     runs-on: ubuntu-latest
+    needs: release
+    env:
+      NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
       - uses: actions/checkout@v2
 
-      - name: Configure Artifactory publish credentials
-        run: |
-          # this sets up credentials to publish to artifactory
-          curl -udeploy-user:${{ secrets.PUBLIC_ARTIFACTORY_DEPLOY_USER_MAGIC_PASSTOKEN }} https://momento.jfrog.io/artifactory/api/npm/npm-public/auth/momento >> ~/.npmrc
-
       - name: Install Node
         uses: actions/setup-node@v1
         with:
           node-version: 14
 
+      - name: Configure Artifactory credentials
+        run: |
+          # lel this is how jfrog wants you to come up with your scoped npm configuration
+          echo @momento:registry=https://momento.jfrog.io/artifactory/api/npm/npm-public/ >> ~/.npmrc
```

Comment:
I left this here so that the action does `npm ci` and tries to install `@momento/wire-types-javascript` we can install it. 
That package hasn't been published to the regular npm and is still in Artifactory.